content/en/security_platform/application_security/setup_and_configure.md

```diff
@@ -292,11 +292,51 @@ if span, ok := tracer.SpanFromContext(request.Context()); ok {
 
 {{< programming-lang lang="ruby" >}}
 
+Use either API to add user information to a trace so that you can monitor authenticated requests in the application.
+
+{{< tabs >}}
+
+{{% tab "Using `Datadog::Kit::Identity.set_user` %}}
+
+Starting with `ddtrace` 1.1.0, a convenience `set_user` method is available:
+
+```ruby
+# Get the active trace
+trace = Datadog::Tracing.active_trace
+
+# Set mandatory user id tag
+Datadog::Kit::Identity.set_user(trace, id: 'd131dd02c56eeec4')
```

Comment:
Not related to this PR, but wouldn't it make sense to mixin `Datadog::Kit::Identity` into `TraceOperation` to allow for `trace.set_user(trace, id: 'd131dd02c56eeec4')`?

Reply:
That might be an option users have to do manually. I'm not sure we want that automatically.